internal/saucecloud/cloud.go

```diff
@@ -485,9 +485,20 @@ func (r *CloudRunner) logSuiteConsole(res result) {
 		return
 	}
 
+	var hasError bool
```

Comment:
I think you can just rely on `res.job.Passed`, no?

Reply:
oh you're right!